main.nf

```diff
@@ -14,7 +14,7 @@ nextflow.preview.dsl=2
 date = new Date().format( 'yyyyMMdd' )
 params.debug = false
 params.email = ""
-params.reference = "${workflow.projectDir}/WS245/WS245.fa.gz"
+params.reference = ""
 parse_conda_software = file("${workflow.projectDir}/scripts/parse_conda_software.awk")
```

Comment:
You can add `checkIfExists: true` for extra safety here.

Reply:
I thought `checkIfExists: true` is an option for channels. Here reference is only a parameter, does it apply too?